src/select/__tests__/select-controlref.test.js

```diff
@@ -73,21 +73,61 @@ describe('setDropdownOpen', function () {
 
     expect(container.querySelectorAll('li').length).toBe(0);
 
-    if (methodsRef.current !== null && methodsRef.current.setDropdownOpen) {
-      methodsRef.current.setDropdownOpen(true);
+    if (controlRef.current !== null && controlRef.current.setDropdownOpen) {
+      controlRef.current.setDropdownOpen(true);
       expect(container.querySelectorAll('li').length).toBe(3);
 
-      methodsRef.current && methodsRef.current.setDropdownOpen(false);
+      controlRef.current && controlRef.current.setDropdownOpen(false);
       expect(container.querySelectorAll('li').length).toBe(0);
 
-      methodsRef.current && methodsRef.current.setDropdownOpen(false);
+      controlRef.current && controlRef.current.setDropdownOpen(false);
       expect(container.querySelectorAll('li').length).toBe(0);
 
       fireEvent.click(screen.getByText('Select...'));
       expect(container.querySelectorAll('li').length).toBe(3);
 
-      methodsRef.current && methodsRef.current.setDropdownOpen(false);
+      controlRef.current && controlRef.current.setDropdownOpen(false);
       expect(container.querySelectorAll('li').length).toBe(0);
     }
   });
 });
+
+describe('setInputValue', function () {
+  it('correctly sets the input value', () => {
+    const options = [
+      { id: 'a', label: 'dragons' },
+      { id: 'b', label: 'unicorns' },
+      { id: 'c', label: 'elves' },
+    ];
+    const methodsRef = React.createRef();
```

Comment:
Is this test still valid or useful? methodsRef looks like it's removed.

Reply:
Sorry that was sloppy. `methodsRef` should be `controlRef` here. I'm confused about why the test did not fail.